### PR TITLE
Jennyf/update lab api 

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabAuthenticationHelper.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        private static string s_secret;
+        private static ClientAssertionCertificate _assertionCert;
+        private static AuthenticationResult _authResult;
+
+        static LabAuthenticationHelper()
+        {
+            //The data.txt is a place holder for the keyvault secret. It will only be written to during build time when testing appcenter.
+            //After the tests are finished in appcenter, the file will be deleted from the appcenter servers.
+            //The file will then be deleted locally Via VSTS task.
+            if (File.Exists(DataFileName))
```
Comment (bgavrilMS) [inline tests/Test.Microsoft.Identity.LabInfrastructure/LabAuthenticationHelper.cs:37]: Do we still do this ? I don't think so, but may be wrong... #Resolved

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs [hunk 2]
```diff
@@ -1,70 +1,58 @@
...
-        [JsonProperty("Scope")]
-        public IList<string> Scopes { get; set; }
+    public class LabApp
+    {
+        [JsonProperty("appid")]
+        public string AppId { get; set; }
 
-        [JsonProperty("Users")]
-        public LabUser User { get; set; }
+        // TODO: this is a list, but lab sends a string. Not used today, discuss with lab to return a list
```
Comment (bgavrilMS) [inline tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs:25]: I think they fixed this. #Resolved

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabAuthenticationHelper.cs (hunk 1 above)
Comment (jennyf19) [reply tests/Test.Microsoft.Identity.LabInfrastructure/LabAuthenticationHelper.cs:37]: no idea. i'll take a look tomorrow. #Resolved

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs [hunk 3]
```diff
@@ -1,70 +1,58 @@
...
+        [JsonProperty("redirecturi")]
+        public string RedirectUri { get; set; }
+
+        [JsonProperty("signinaudience")]
+        public string Audience { get; set; }
+
+        // TODO: this is a list, but lab sends a string. Not used today, discuss with lab to return a list
```
Comment (bgavrilMS) [inline tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs:32]: And this. #WontFix

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs (hunk 2 above)
Comment (jennyf19) [reply tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs:25]: we have the same comment in msal. i'll check on this. #WontFix

**Review by bgavrilMS — APPROVED** [review]

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs (hunk 2 above)
Comment (jennyf19) [reply tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs:25]: looks to be a string still. and they are heads down on security stuff. i'll follow up next week.

---
In reply to: [355231326](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1681#discussion_r355231326) [](ancestors = 355231326)

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabAuthenticationHelper.cs (hunk 1 above)
Comment (jennyf19) [reply tests/Test.Microsoft.Identity.LabInfrastructure/LabAuthenticationHelper.cs:37]: this is needed.

File: tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs (hunk 2 above)
Comment (bgavrilMS) [reply tests/Test.Microsoft.Identity.LabInfrastructure/LabResponse.cs:25]: Thx for following up. I think they have a work item on this already.

**Pull request merged**